thevoid/connection.cpp

```diff
@@ -796,6 +873,15 @@ void connection<T>::process_data()
 				send_error(http_response::bad_request);
 				return;
 			} else {
+				CONNECTION_INFO(
+					"received new request: method: %s, url: %s, local: %s, remote: %s, headers: %s",
```

Comment:
How can I disable header dump?
I do know browser requests with tens of kilobytes of headers, this will kill the whole server performance.